Change View of accounts into ListView

MBlokhuijzen opened the pull request.

willlunniss commented:
What testing have you done on this (Android Device/number of accounts)?

Would you consider switching straight to RecyclerView instead to give a better layout on tablets? 

@alorma was looking at doing some re-working, but haven't heard from him in a while.

MBlokhuijzen commented:
Tested it on a Nexus 5 with 2 Accounts.
Removed the ScrollView, it seemed odd to be on top of the layout instead around the ListView

Could try to change it into a RecyclerView, I didn't test it on a tablet.

nelenkov commented:
Could you make the diff readable? Not sure what you changed, but it looks like the whole activity class has been replaced, which is not the case. Also, `CheckboxListener` is not really a meaningful class name, consider renaming (e.g., `AccountSelectedListener` or some such).

MBlokhuijzen commented:
Not sure how to make it readable. Screwed up the indenting. Changed it to a RecyclerView and renamed the Listener, thanks for the tip.

Tested on the same device.

willlunniss commented:
Just tested on my Nexus 5 running 6.0 and the apps disappear when refreshing

As for the formatting, convert the double spaces back to tabs and it should show that you have only changed part of the class
